src/open_inwoner/scss/components/Dropdown/Dropdown.scss

```diff
@@ -16,6 +16,34 @@
     background-color: var(--color-white);
     border: 1px solid var(--color-mute);
     z-index: 1;
+
+    &::before {
+      border-bottom: 6px solid var(--color-mute);
+      border-left: 6px solid var(--color-white);
+      border-right: 6px solid var(--color-white);
+      box-sizing: border-box;
+      content: '';
+      display: block;
+      height: 6px;
+      right: 0.4em;
+      position: absolute;
+      top: -7px;
+      width: 12px;
+    }
+
+    &::after {
+      border-bottom: 6px solid var(--color-white);
+      border-left: 6px solid transparent;
+      border-right: 6px solid transparent;
+      box-sizing: border-box;
+      content: '';
+      display: block;
+      height: 6px;
+      right: 0.4em;
+      position: absolute;
+      top: -6px;
+      width: 11px;
+    }
```

Comment:
Why the 1px difference between before/after for `top`/`width`?

Reply:
@pi-sigma Ha! Well seen - I will remove both widths  - i just copied my own code from the navigation menus in order to create the two overlapping triangles.